lib/ssh_scan/database/mongo_queue.rb

```diff
@@ -0,0 +1,55 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module SSHScan
+  module QueueDB
+    class MongoDb
+      attr_reader :queue_collection
```

Comment:
I think we'd be better off here if we leveraged the pre-existing DB classes and made JobQueue just use the high-level DB abstraction.